executor/set.go

```diff
@@ -178,7 +178,7 @@ func (e *SetExecutor) setSysVariable(name string, v *expression.VarAssignment) e
 			valStr, err = value.ToString()
 			terror.Log(errors.Trace(err))
 		}
-		log.Infof("[con:%d] set system variable %s = %s", sessionVars.ConnectionID, name, valStr)
+		log.Infof("con:%d %s=%s", sessionVars.ConnectionID, name, valStr)
```

Comment:
There are places where `con:` is not enclosed by '[]'.
How about we should remove all the `[]` that enclosing `con:`?

Reply:
should we more shorter use `c:` instead of `conn:`?

and I have a idea: maybe we encapsulation all log method and in `ctx idioms`, in the method body it use prealloc log header in thread load level(e.g. `"c:123, tx:12, tr:33"`), then need for every place to to fomat conn mannul, and no need to alloc header part every time?